### PR TITLE
CLOUD-597 Script - Change service bus connection string

### PR DESCRIPTION
**Pull request by soniadunkley**

### CODE REVIEW

File: ReplaceServiceBusConnectionString.ps1 [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+$configFiles = Get-ChildItem -Path $directoryPath -Recurse  -Include *.config
+$configFiles | ForEach-Object {
+        $configFile = $_.FullName
+
+        $xmlFile = (Get-Content $configFile) -as [Xml]
+
+        foreach($add in $xmlFile.configuration.appSettings.add) {
+            if ($add.key -eq 'ServiceBus') 
+            {                                
+                write-host "File Changed : " $configFile
```
Comment (margani) [inline ReplaceServiceBusConnectionString.ps1]: it's good to print "File Changed" after actually changing it.

Comment (margani) [inline ReplaceServiceBusConnectionString.ps1]: it's good to print "File Changed" after actually changing it.

File: ReplaceServiceBusConnectionString.ps1 [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+$jsonFiles = Get-ChildItem -Path $directoryPath -Recurse  -Include *configuration.json
+$jsonFiles | ForEach-Object {
+        $jsonFile = $_.FullName
+                  
+        $jsonObject = (Get-Content $jsonFile) | ConvertFrom-Json
+
+        if (Get-Member -InputObject $jsonObject -Name "ServiceBus" -MemberType Properties)
+        {
+            write-host "File Changed : " $jsonFile
```
Comment (margani) [inline ReplaceServiceBusConnectionString.ps1]: same here

File: ReplaceServiceBusConnectionString.ps1 (hunk 1 above)
Comment (soniadunkley) [reply ReplaceServiceBusConnectionString.ps1]: done

File: ReplaceServiceBusConnectionString.ps1 (hunk 2 above)
Comment (soniadunkley) [reply ReplaceServiceBusConnectionString.ps1]: done

**Review by margani — APPROVED** [review]

**Review by ma499 — CHANGES_REQUESTED** [review]
Can we implement Component-Based help by adding documentation in the correct comment format? See https://technet.microsoft.com/en-us/library/ff458353.aspx

**Review by rickyattiliaaq — DISMISSED** [review]

File: ReplaceServiceBusConnectionString.ps1 [hunk 3]
```diff
@@ -0,0 +1,62 @@
+<#
+
+.SYNOPSIS
+
+Allows you to update service bus connection strings in the directory (and sub-directories) supplied.
+
+.DESCRIPTION
+
+Updates the service connection string in .config and .json files
```
Comment (ma499) [inline ReplaceServiceBusConnectionString.ps1:10]: Can we add the Syntax and an Example?

Comment (soniadunkley) [reply ReplaceServiceBusConnectionString.ps1:10]: If you run it, there's default syntax which I believe is enough of an example. But if you want more of an example I could do that, let me know

**Review by ma499 — APPROVED** [review]

**Comment by govinduv** [comment]
@soniadunkley   can you please  add  EXAMPLE to the script ( for  example  look at  https://github.com/Aqovia/dev-scripts/blob/master/aqovia-nuget-update.ps1 )

**Pull request merged**